packages/botonic-react/src/webchat-app.jsx

```diff
@@ -286,9 +286,9 @@ export class WebchatApp {
     enableAnimations = enableAnimations || this.enableAnimations
     defaultDelay = defaultDelay || this.defaultDelay
     defaultTyping = defaultTyping || this.defaultTyping
-    storage = storage || this.storage
-    storageKey = storageKey || this.storageKey
     server = server || this.server
+    this.storage = storage || this.storage
+    this.storageKey = storageKey || this.storageKey
```

Comment:
The props of `storage` and `storageKey` sent to the `Webchat` component have to be changed to `this.storage` and `this.storageKey`, otherwise no storage will be created unless stated at runtime.